index.html

```diff
@@ -399,6 +414,18 @@ <h2 class='mb-0'>Gato-Bots</h2>
                       <rect class='fill' fill='#fff' x='210' y='160' width='20' height='20' />
                     </svg>
                   </div>
+                  <div class='carousel-item'>
+                    <svg viewBox='0 80 320 100' width='100%'>
+                      <rect class='fill' fill='#fff' x='135' y='80' width='50' height='19' />
+                      <rect class='fill' fill='#fff' x='130' y='100' width='60' height='60' />
+
+                      <path class='stroke-brass' d='M 130 90 C 110 100, 100 130, 100 140' stroke='#fff' stroke-width='4' fill='none' />
+                      <rect class='fill' fill='#fff' x='85' y='150' width='30' height='30' />
+
+                      <path class='stroke-brass' d='M 190 90 C 210 100, 220 130, 220 140' stroke='#fff' stroke-width='4' fill='none' />
+                      <rect class='fill' fill='#fff' x='205' y='150' width='30' height='30' />
+                    </svg>
+                  </div>
```

Comment:
same kinda deal with the new body.

the first `rect`s are the two body segments, the `path` + `rect`s the arms plus bigger hands